src/mlpack/tests/init_rules_test.cpp

```diff
@@ -123,4 +127,37 @@ BOOST_AUTO_TEST_CASE(OivsInitTest)
   BOOST_REQUIRE_EQUAL(1, 1);
 }
 
+// Test the GaussianInitialization class.
+BOOST_AUTO_TEST_CASE(GaussianInitTest)
+{
+  arma::mat weights;
+  arma::cube weights3d;
+  const size_t row = 7;
+  const size_t col = 7;
+  const size_t slice = 2;
+  double mean, mean3d;
+  double var,var3d;
+  mean=mean3d=1;
+  var=var3d=1;
+  GaussianInitialization t(0, 0.2);
+  RandomSeed(21);
```

Comment:
Nevermind you can't since GaussianDistribution provides the option to use a custom covariance matrix. So Do you think we could remove the random seed and let the test run for e.g. 5 trails and if it passes one from the 5 trails it's fine?